app/__init__.py

```
@@ -12,18 +12,20 @@ def create_app():
 
-    # ✅ Initialize MongoDB
+    # MongoDB connection
     connect(host=app.config['MONGODB_SETTINGS']['host'])
 
-    # ✅ CORS and JWT
-    CORS(app, supports_credentials=True, origins="*")
+    # ✅ Allow all origins
+    CORS(app, resources={r"/*": {"origins": "*"}}, supports_credentials=True)
+
+    # JWT Setup (reads expiry values from config)
     jwt = JWTManager(app)
```

Ruff (0.12.2)

Local variable `jwt` is assigned to but never used

Remove assignment to unused variable `jwt`

(F841)